lfucg-exactions/client/js/components/AccountLedgerForm.js

```diff
@@ -474,7 +486,7 @@ function mapDispatchToProps(dispatch, params) {
                         [field_show]: value,
                     };
                     dispatch(formUpdate(account_update));
-                    if (field === 'account_from' && !value_name.includes('LFUCG')) {
+                    if (field === 'account_from' && (value_name.indexOf('LFUCG') === -1)) {
```

Comment:
Just curious what the benefit of switching from `includes` to `indexOf` is?

Reply:
includes does not work on Internet Explorer, indexOf does.